tensorflow/python/keras/backend_test.py

```diff
@@ -1880,5 +1880,33 @@ def test_get_session_different_graphs(self):
       self.assertIsNot(session, keras.backend.get_session())
 
 
+class ControlOpsTests(test.TestCase):
+
+  @test_util.run_all_in_graph_and_eager_modes
+  def test_function_switch_basics(self):
+    pred = keras.backend.placeholder([], dtype='bool')
+
+    tensor = keras.backend.switch(pred, lambda: array_ops.constant(True),
+                                  lambda: array_ops.constant(False))
+    value = keras.backend.function(pred, tensor)([True])
+    self.assertEqual(True, value)
```

Comment:
This can be replaced with self.assertTrue(value) which is more readable.

Reply:
Sure, this is done as suggested by you.

Regards
Amit